android/sdl_android/src/androidTest/java/com/smartdevicelink/managers/ManagerUtilityTests.java

```diff
@@ -0,0 +1,206 @@
+		assertFalse(ManagerUtility.WindowCapabilityUtility.hasImageFieldOfName(capability, ImageFieldName.alertIcon));
+
+		imageFieldList.add(new ImageField(ImageFieldName.alertIcon, allImageFileTypes));
+		capability.setImageFields(imageFieldList);
+		assertTrue(ManagerUtility.WindowCapabilityUtility.hasImageFieldOfName(capability, ImageFieldName.graphic));
+		assertTrue(ManagerUtility.WindowCapabilityUtility.hasImageFieldOfName(capability, ImageFieldName.alertIcon));;
+
+		imageFieldList.clear();
+		imageFieldList.add(null);
+		capability.setImageFields(imageFieldList);
+		assertFalse(ManagerUtility.WindowCapabilityUtility.hasImageFieldOfName(capability, ImageFieldName.graphic));
+		assertFalse(ManagerUtility.WindowCapabilityUtility.hasImageFieldOfName(capability, ImageFieldName.alertIcon));
+
+		imageFieldList.add(new ImageField(ImageFieldName.alertIcon, allImageFileTypes));
+		capability.setImageFields(imageFieldList);
+		assertFalse(ManagerUtility.WindowCapabilityUtility.hasImageFieldOfName(capability, ImageFieldName.graphic));
+		assertTrue(ManagerUtility.WindowCapabilityUtility.hasImageFieldOfName(capability, ImageFieldName.alertIcon));
+
+	}
+
+
+	@Test
+	public void testGetMaxNumberOfMainFieldLines() {
+
+		WindowCapability capability = new WindowCapability();
+		capability.setTextFields(ManagerUtility.WindowCapabilityUtility.getAllTextFields());
+
+		int maxNumerOfLines = ManagerUtility.WindowCapabilityUtility.getMaxNumberOfMainFieldLines(capability);
+
+		assertEquals(4, maxNumerOfLines);
+
+		//Single line
+		List<TextField> singleLineList = new ArrayList<>();
+		singleLineList.add(new TextField(TextFieldName.mainField1, CharacterSet.UTF_8, 500, 8));
+		capability.setTextFields(singleLineList);
+		maxNumerOfLines = ManagerUtility.WindowCapabilityUtility.getMaxNumberOfMainFieldLines(capability);
+		assertEquals(1, maxNumerOfLines);
+
+		singleLineList.add(new TextField(TextFieldName.mainField2, CharacterSet.UTF_8, 500, 8));
+		capability.setTextFields(singleLineList);
+		maxNumerOfLines = ManagerUtility.WindowCapabilityUtility.getMaxNumberOfMainFieldLines(capability);
+		assertEquals(2, maxNumerOfLines);
+
+		singleLineList.add(new TextField(TextFieldName.mainField3, CharacterSet.UTF_8, 500, 8));
+		capability.setTextFields(singleLineList);
+		maxNumerOfLines = ManagerUtility.WindowCapabilityUtility.getMaxNumberOfMainFieldLines(capability);
+		assertEquals(3, maxNumerOfLines);
+
+		singleLineList.add(new TextField(TextFieldName.mainField4, CharacterSet.UTF_8, 500, 8));
+		capability.setTextFields(singleLineList);
+		maxNumerOfLines = ManagerUtility.WindowCapabilityUtility.getMaxNumberOfMainFieldLines(capability);
+		assertEquals(4, maxNumerOfLines);
+
+		List<TextField> nullList = new ArrayList<>();
+		nullList.add(null);
+		assertNotNull(nullList);
+		capability.setTextFields(nullList);
+		assertNotNull(capability);
+		assertNotNull(capability.getTextFields()); //Fails
```

Comment:
This passes after the fix, right? can we remove the comment if it is not relevant anymore?